public/alert.css

```diff
@@ -0,0 +1,47 @@
+alert {
+    display: block;
+    background-color: white;
+    color: black;
+    max-width: 800px;
+    width: 100%;
+    padding: 10px;
+    border-left: 7px solid var(--red);
+    /*box-shadow is from https://www.w3schools.com/w3css/4/w3.css*/
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2),0 6px 20px 0 rgba(0,0,0,0.19);
+    border-radius: 4px;
+    position: relative;
+    top: 15px;
+}
+alert .message {
+    display: inline-block;
+    padding-top: 5px;
+}
+alert .right {
+    float: right;
+}
+alert .remove {
+    font-size: 25px;
+    margin-left: 20px;
+    display: inline-block;
+    align-items: center;
+    text-align: center;
+    overflow: visible;
+    cursor: pointer;
+    transition: 0.2s;
+}
+alert .remove:hover {
+    color: var(--red);
+}
+alert.animateOut {
+    animation: 0.4s linear animateout;
+}
+@keyframes animateout{
```

Comment:
It's worth noting that Svelte already has pretty great built-in animation support – see [this demo](https://svelte.technology/repl?version=2.16.1&demo=transitions-in-out)